standards/version-control.md

```diff
@@ -0,0 +1,41 @@
+# Version Control
+These standards are based on Defra Digital's [version control standards](https://github.com/DEFRA/software-development-standards/blob/master/standards/version_control_standards.md), but are extended to cover not only the application version, but version control of Docker images and Helm charts.
+
+## Semantic Versioning
+We use semantic versioning to manage code, images and charts.
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+- MAJOR version when you make incompatible API changes,
+- MINOR version when you add functionality in a backwards compatible manner, and
+- PATCH version when you make backwards compatible bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+
+Full details can be found at [semver.org](https://semver.org).
+
+## Application
+- for a Node.js microservice, the version is updated in the `package.json` file's `version` property.  
```

Comment:
Not really anything to do with this PR but it has reminded me. I think we should extend the version increment to include `package-lock.json` so they are kept in sync. Atm, we have a drift between files, which isn't a problem but when packages are updated the `package-lock.json` gets updated with what the current version is and depending when that happens it might be the very latest version or it might end up being the previous version. If we included the file in the checks, it would force the update.
Just an idea...

Reply:
What would be the benefit of keeping lock file in sync?  Is it good practice or would it potentially cause an issue if we don't?

Reply:
A cursory Google doesn't find anything interesting.
IME there are no problems with it being out of sync and is quite common for that to be the case. It is mostly the case for our repos and it hasn't caused any issues.
My preference is to have them in sync and I tend to do this but that is about the extent of doing it.

Reply:
Maybe we should leave that to the consuming teams to decide or not rather than enforcing it by default?